src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="container"
+    >
```

Comment:
Remember to keep your attributes correctly formatted. If the HTML element has a number of attributes more than 2, start each one, including the first, on a new line with 2-space indentation related to the tag. The tag’s closing bracket should be on the same level as the opening one.